plugins/keycloak-backend/src/module/catalogModuleKeycloakEntityProvider.ts

```diff
@@ -59,13 +60,14 @@ export const catalogModuleKeycloakEntityProvider = createBackendModule({
       },
       async init({ catalog, config, logger, scheduler }) {
         catalog.addEntityProvider(
-          KeycloakOrgEntityProvider.fromConfig(config, {
-            id: 'development',
-            logger,
-            scheduler,
-            userTransformer: userTransformer,
-            groupTransformer: groupTransformer,
-          }),
+          KeycloakOrgEntityProvider.fromConfig(
+            { config, logger },
+            {
+              scheduler: scheduler,
+              userTransformer: userTransformer,
```

Comment:
Do we want to add here default schedule @schultzp2020?  And update docs accordingly.

```suggestion
              scheduler: scheduler,
              schedule: scheduler.createScheduledTaskRunner({
                frequency: { minutes: 30 },
                timeout: { minutes: 3 },
              }),
              userTransformer: userTransformer,
```

Reply:
Yes

Reply:
Yes